### PR TITLE
Feature / Update offer debugger to match JSON schema

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

- Fix creating SWEDISH HOUSE quote.
- Use uppercase quote types.
- Fix NO/DK/SE quote inputs to match JSON schema
- Handle UW limit errors in debugger.

## Why?

Some updates have landed in Underwriter so it was time to make the same changes here.

**Ticket(s): []**

### CODE REVIEW

**Review by guilhermespopolin — APPROVED** [review]
LGTM. Tested locally and I was able to create quote bundles.

**Comment by amandaflod** [comment]
Good stuff! Liked the renaming you did, makes things much clearer :D I was still not able to create a house quote when testing in the review app, it says Swedish house in the input, but the response says something different 🤔 Let me know if I'm missing something here!

Screenshot 2021-12-08 at 16 34 46

**Comment by amandaflod** [comment]
Good stuff! Liked the renaming that you did in a few places, makes things much clearer 😄 Only thing is that I was not able to create a house quote in the review app, it says house in the input but the response says something different 🤔 Let me know if I'm missing something here!
Screenshot 2021-12-08 at 16 34 46

**Comment by styrelius** [comment]
> Good stuff! Liked the renaming that you did in a few places, makes things much clearer 😄 Only thing is that I was not able to create a house quote in the review app, it says house in the input but the response says something different 🤔 Let me know if I'm missing something here!

@amandaflod  The review app for the other offer page PR doesn't have these changes - but when I run this locally I get the correct type of quote for house :)

image

**Comment by styrelius** [comment]
Haha sorry about the crazy size of that image 😳

File: src/client/pages/OfferDebugger/components/QuoteData.tsx [hunk 1]
```diff
@@ -160,13 +162,30 @@ const getSwedishAccidentQuoteValues = (values: any) => {
...
+  return {
+    ...holderData,
+    data: {
+      ...quoteTypeValues,
+      type: QuoteType.SwedishHouse,
+      ancillaryArea: 0,
+      yearOfConstruction: 1977,
+      numberOfBathrooms: 2,
+      extraBuildings: [],
+      isSubleted: false,
```
Comment (styrelius) [inline src/client/pages/OfferDebugger/components/QuoteData.tsx:184]: We should probably add all these things to the `QuoteCart` query - currently we only ask for `street` back.

**Review by styrelius — COMMENTED** [review]
Nice, works like a charm ✨ Also good renamings!

**Review by styrelius — APPROVED** [review]
Let's fix the query later 🙃

**Pull request merged**